SONAR-netCDF4/docs/tableBeamGroup1.adoc

```diff
@@ -95,6 +111,15 @@
 |:long_name = "Time offset that is subtracted from the timestamp of each sample" | |
 |:units = "s" | |
 | | |
+|float transmit_time_delay(ping_time, tx_beam) |O |Time delay for each transmit beam between the ping time and the transmit time
+|:long_name = "Time delay for each transmit beam between the ping time and the transmit time" | |
```

Comment:
For this and blanking_interval there should be an equation to make it very clear how these parameters should be used.

Reply:
So you are suggesting to not have transmit_time_delay, but use the existing sample_time_offset instead and then add in blanking_interval? 

Seems ok, but is there a good reason to have both a blanking_interval and transmit_time_delay?

Reply:
Forgot to mention: Equation 2 (in section 3.1) specifies how to calculate range, so once we agree on these delay variables, we need to update equation 2.

Reply:
The transmit time delay is the only thing that is given to me for EM sounders. The main reason why I wanted to keep it like this is because we need to make a distinction between the tx time offset and the Tx/Rx time offset if we want to raytrace and compute precisely the position of sample (for bathymetry mainly). (https://hydrography.ca/wp-content/uploads/files/2014conference/20-Hamilton-et-al-Algorithm-for-non-concentric-MB-array-geometry.pdf)
I agree that it is becoming confusing, I do not really known  any  algorithm giving the same result as the constructor. That's why we will add this variable only in the vendor group and remove it from the format.

blanking interval is just the number of samples that are not recorded in the file (to prevent saturation near the transducer or to save space on disk if we are not interested in watercolumn near to the surface). We will correct the equation 2 (and divide our celerity by 2 ;-) )